jplag.frontend.golang/src/test/java/de/jplag/golang/GoFrontendTest.java

```diff
@@ -158,7 +157,7 @@ private List<Integer> getCodeLines(List<String> lines) {
      * @param fileName The file name of the complete code example
      */
     private void testTokenCoverage(TokenList tokens, String fileName) {
-        var foundTokens = StreamSupport.stream(tokens.allTokens().spliterator(), true).mapToInt(Token::getType).sorted().distinct().boxed().toList();
+        var foundTokens = tokens.stream().parallel().mapToInt(Token::getType).sorted().distinct().boxed().toList();
```

Comment:
Why parallel? Is there really a performance issue here?

Reply:
The only reason is that it was set to parallel before, and that the order does not matter here. But it is surely not necessary performance-wise.

Reply:
Ah, I missed that, then it is a mistake that was already in there. Let's remove `parallel()` then.